src/coreclr/jit/emitxarch.cpp

```diff
@@ -34,11 +34,22 @@ bool emitter::IsSSEOrAVXInstruction(instruction ins)
     return (ins >= INS_FIRST_SSE_INSTRUCTION) && (ins <= INS_LAST_AVX_INSTRUCTION);
 }
 
+bool emitter::IsSSEOrAVXorAVX512Instruction(instruction ins)
```

Comment:
It's overall inconsistent due to the age of the codebase, but we aim for new methods to all be prefixed with a header of the form:
```
//------------------------------------------------------------------------
// MethodName: ...
// ...
//
// Arguments:
//    param1 - ...
//    param2 - ...
//
// Return Value:
//    ...
```

Reply:
Added comments.